tn_api_hotel.yml

```diff
@@ -239,89 +312,581 @@ components:
+                                example: 2G3C
+                            provider:
+                                description: Fare provider to use.<br  />“1V” - Apollo, “1G” - Galileo, “1P” - Worldspan, “1A” - Amadeus
+                                required: true
+                                type: string
+                                example: 1V
+                            pcc_currency:
+                                description: The currency that is associated with the PCC. This will be the currency that any bookings are made and charged in.
+                                type: string
+                                example: CAD
+                      conversion_rate:
+                          type: float
+                          description: The conversion rate to get the price in the currency requested.
+                          example: 0.45
+          hotel_itinerary_reference:
+              type: string
+              example: '1'
+              description: A reference that indicates the itineraries position in the flow of itineraries in the super_trip with the index starting at 0.
+          next_result_reference:
+              type: string
+              example: 'GvCe60ac0/ca8J7rRpzT99+A990Uzoz9uPnDvrR9vlPYyiAdwA/4TedEKvTDZyf87Yyw51a62Z0vr9lr1UD15D2BfyUpgIOXDnCDubYq+Qg9pzbZYLqUbA=='
+              description: The reference used to get the next list of results from the GDS.
 
     HotelSearchRequest:
       required:
-        - super_trip_id
+          - location
+          - check_in_date
+          - check_out_date
+          - hotel_itinerary_reference
+          - credential_info
+          - total_travellers
+          - total_rooms
       properties:
           super_trip_id:
               type: string
               example: "bdf876sf6sfsdf87sdf"
-              description: id of the super trip
-          iata:
+              description: id of the super trip, may not be present in the initial search.
+          location:
               type: string
               example: "YYZ"
-              description: location(city) to search hotels in
+              description: location (city iata) to search for hotels in.
           check_in_date:
               type: string
               format: date
-              description: Date of check-in to search hotel availibilities on.
+              description: "Date of check-in. format: YYYY-MM-DD. Any trailing characters after do not affect the API’s functionality."
           check_out_date:
               type: string
               format: date
-              description: Date to checkout from the hotel.
+              description: "Date to checkout from the hotel. format: YYYY-MM-DD. Any trailing characters after do not affect the API’s functionality."
+          currency:
+              type: string
+              example: 'CAD'
+              description: Currency to make the search in.
+          total_travellers:
```

Comment:
On MCFS it seems like its expecting this value instead of total_travellers: 
![image](https://user-images.githubusercontent.com/13696086/107275578-f3b22180-6a1f-11eb-87a3-dd787ec7eb6c.png)

Reply:
fixed - I changed the name to traveller_total